tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -186,6 +187,21 @@ Verify JupyterHub Pod Logs Dont Have Errors About Distutil Library
     ...       ODS-586
     Verify Errors In Jupyterhub Logs
 
+Verify Grafana Is Connected To Prometheus Using TLS
+    [Documentation]    Verifies Grafana is connected to Prometheus using TLS
+    [Tags]    Tier2
+    ...       ODS-963
+    [Setup]  Set Library Search Order  Selenium Library
+    Verify Grafana Datasources Have TLS Enabled
+    ${grafana_url} =  Get Grafana URL
```

Comment:
Lines 196-201 could be wrapped in a keyword  `Verify Grafana Can Obtain Data From Prometheus Datasource`